codecov/__init__.py

```diff
@@ -376,6 +376,9 @@ def main(*argv, **kwargs):
     gcov.add_argument(
         "--gcov-exec", default="gcov", help="gcov executable to run. Defaults to 'gcov'"
     )
+    gcov.add_argument(
+        "--gcov-out", action="store_false", default=True, help="Disable gcov output"
```

Comment:
I could be wrong, but I don't think we need to specify `default=True` if `action` is set here. Also, I think the booleans are flipped based on the helptext and what exists on the bash uploader.

Reply:
I know this isn't what we do the bash uploader, and I plan to change it, but can we name this something more obvious like `--silence-gcov`?

Reply:
I agree that the naming of the argument could be better. How about `--no-gcov-out`?

Reply:
works for me!

Reply:
Fixed.